### ISSUE TITLE
plugin not working in current browser/configuration

### ISSUE THREAD

**Issue by hexwren** [lawrencehook/remove-youtube-suggestions#42]
everyone and their grandmother appear to be dropping support for waterfox classic, so if it's no longer supported by this project as well, I get it, but I'd like to check that. I'm currently running WFC 2021.10 (64) and just the other day, RYS stopped blocking the indicated portions of the YT layout for me. I initially assumed that an update to YT's page format was behind it and checked for an update here. Seeing none, I eventually checked to see if the plugin was still working on chrome (which I keep around for in-case-shit reasons) and it was just fine.

Which brings us up to date. Let me know if there's anything I can specifically troubleshoot about this issue or any more info I can provide.

**Comment by lawrencehook** [comment]
Hi hexwren, sorry to hear RYS has stopped working for you.

I'm not actually familiar with Waterfox at all. Is it a firefox alternative?

If I get a chance I'll do some digging.

**Comment by hexwren** [comment]
yeah, waterfox got forked from firefox way back when, and I hopped to it
because it had a 64-bit version when firefox didn't have one yet. muddying
the waters further is that WF itself forked again into WF and WF classic,
with WF abandoning its old code and compatibility for easier dev upkeep,
which I certainly understand.

**Comment by remi-garcia** [comment]
If I remember well, when Firefox changed to Firefox Quantum (2017), Waterfox forked and only kept the security update.
